package.json

```diff
@@ -147,6 +152,7 @@
     "classnames": "~2.2.6",
     "crypto-js": "~3.3.0",
     "dotenv": "^10.0.0",
+    "har-validator": "^5.1.5",
```

Comment:
We shouldn't be adding this as a dependency. We should wait for the cypress fix.

Reply:
done